libraries/AdaptiveExpressions/Options.cs

```diff
@@ -24,5 +27,13 @@ public Options(Options opt)
         /// The function delegate.
         /// </value>
         public Func<string, object> NullSubstitution { get; set; } = null;
+
+        /// <summary>
+        /// Gets or sets a value, a locale of CultureInfo.
+        /// </summary>
+        /// <value>
+        /// The locale info.
+        /// </value>
+        public string Locale { get; set; } = Thread.CurrentThread.CurrentCulture.Name;
```

Comment:
>= Thread.CurrentThread.CurrentCulture.Name; [](start = 43, length = 43)

Won't this always be overridden by the constructor?  Either one or the other--I'd prefer the constructor.

Reply:
put the initilization in constructor. Removed it in gets and sets